product_docs/docs/tpa/23/architecture-PGD-Always-ON.mdx

```diff
@@ -5,11 +5,17 @@ originalFilePath: architecture-PGD-Always-ON.md
 
 ---
 
+!!!Note
+
+This architecture is for Postgres Distributed 5 only. 
+If you require PGD 4 or 3.7 please use [BDR-Always-ON](architecture BDR-Always-ON/).
```

Comment:
Link is not displaying. I suspect that since it is nested in a note, the link will have to be in HTML instead of markdown.